src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -71,13 +71,14 @@ export function hyperLinkFormatter(row: number | undefined, cell: any | undefine
 		valueToDisplay = 'NULL';
 		if (!value.isNull) {
 			cellClasses += ' xmlLink';
-			valueToDisplay = escape(value.displayValue);
+			valueToDisplay = getCellDisplayValue(value.displayValue);
 			return `<a class="${cellClasses}">${valueToDisplay}</a>`;
 		} else {
 			cellClasses += ' missing-value';
 		}
 	} else if (isHyperlinkCellValue(value)) {
-		return `<a class="${cellClasses}" title="${escape(value.displayText)}">${escape(value.displayText)}</a>`;
+		valueToDisplay = getCellDisplayValue(value.displayText);
```

Comment:
I assume truncating doesn't change the behavior when clicking the link.  i.e. the value in the new document comes from the underlying data model, not the cell.

Reply:
yes, that is correct.